club/theme.json

```
@@ -350,15 +350,16 @@
                 },
                 "spacing": {
                     "padding": {
-                        "bottom": "var(--wp--custom--gap--horizontal)",
-                        "left": "var(--wp--custom--gap--horizontal)",
-                        "right": "var(--wp--custom--gap--horizontal)",
-                        "top": "var(--wp--custom--gap--horizontal)"
+                        "bottom": "0px",
+                        "left": "0px",
+                        "right": "0px",
+                        "top": "0px"
                     }
                 },
                 "typography": {
-                    "fontSize": "var(--wp--preset--font-size--large)",
-                    "fontStyle": "italic"
+                    "fontWeight": "700",
+                    "fontSize": "1.5rem",
+                    "lineHeight": "1.25"
```

Should this also be 1.3?
```suggestion
                    "lineHeight": "1.3"
```

---

Also, I noticed setting the line height here is not respected by the pullquote. I took a video, the I added the line height to the paragraph element in the inspector to demonstrate:

https://user-images.githubusercontent.com/5375500/184425077-59ff0b62-0ef7-4660-8b0a-4e33abb5656d.mov

---

Yep, there is another Gutenberg PR to solve that: https://github.com/WordPress/gutenberg/pull/43195
It seems simple so maybe it would be merged soon.